<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pbIntroducir">
     <property name="text">
      <string>Introducir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="lePrecio">
     <property name="placeholderText">
      <string>Precio del Instrumento</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="leCantidadTeclas">
     <property name="placeholderText">
      <string>Teclas del Instrumento</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="leCantidadCuerdas">
     <property name="placeholderText">
      <string>Cuerdas del Instrumento</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pbVerStock">
     <property name="text">
      <string>Ver Stock</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="leMetal">
     <property name="placeholderText">
      <string>Metal del Instrumento</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="cbInstrumento">
     <item>
      <property name="text">
       <string>Guitarra</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Viento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Teclado</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="leMarca">
     <property name="placeholderText">
      <string>Marca del Instrumento</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
